### ISSUE TITLE
package: max open files limits are too low by default

### ISSUE THREAD

**Issue by EmilienM** [containers/libpod#1354]
/kind bug

**Description**

When podman package is built with defaults, the open files limits are taken from the host and can be tool low for some containers like memcached, etc.
Note that buildah and docker are built with 1048576 and we should probably change podman defaults to use this number as well.

**Steps to reproduce the issue:**

1. Deploy memcached container with MAXCONN set to 8192.

**Describe the results you received:**

Failure:
`failed to set rlimit for open files. Try starting as root or requesting smaller maxconns value`

**Describe the results you expected:**

No failure.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
Version:       0.8.5-dev
Go Version:    go1.10.3
OS/Arch:       linux/amd64
```

**Output of `podman info`:**

```
host:
  Conmon:
    package: podman-0.8.2.1-1.gitf38eb4f.el7.x86_64
    path: /usr/libexec/podman/conmon
    version: 'conmon version 1.12.0-dev, commit: 74bba30adae4fdcacf887d35e4d6331318765028-dirty'
  MemFree: 210780160
  MemTotal: 8201699328
  OCIRuntime:
    package: runc-1.0.0-37.rc5.dev.gitad0f525.el7.x86_64
    path: /usr/bin/runc
    version: 'runc version spec: 1.0.0'
  SwapFree: 1595658240
  SwapTotal: 1610608640
  arch: amd64
  cpus: 4
  hostname: undercloud75.localdomain
  kernel: 3.10.0-862.11.6.el7.x86_64
  os: linux
  uptime: 26h 37m 53s (Approximately 1.08 days)
insecure registries:
  registries: []
registries:
  registries:
  - docker.io
  - registry.centos.org
  - registry.access.redhat.com
store:
  ContainerStore:
    number: 79
  GraphDriverName: overlay
  GraphOptions:
  - overlay.override_kernel_check=true
  GraphRoot: /var/lib/containers/storage
  GraphStatus:
    Backing Filesystem: xfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
  ImageStore:
    number: 39
  RunRoot: /var/run/containers/storage
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Running everything in Virtualbox.

```
podman run fedora cat /proc/self/limits
Max open files            1024                 1024                 files
 
docker run fedora cat /proc/self/limits
Max open files            1048576              1048576              files
```

**Comment by mheon** [comment]
Opened #1355 to fix

**Comment by mheon** [comment]
Closed via #1355
